src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -271,6 +271,13 @@ final class SecureStore: Store, AntigenTestProfileStoring {
 		set { kvStore["invalidationRulesCache"] = newValue }
 	}
 	
+	// MARK: - Protocol HealthCertificateBoosterNotificationCaching
+	
+	var boosterRulesCache: ValidationRulesCache? {
+		get { kvStore["boosterRulesCache"] as ValidationRulesCache? ?? nil }
+		set { kvStore["boosterRulesCache"] = newValue }
```

Comment:
Could not find in the tech spec something about caching. Did i oversee something? If so, it would explain why you added the 304-http-code to the downloading.

Reply:
Okay we need some caching so my comment can be ignored.